docs/desktop/gnome/decoder.md

```diff
@@ -0,0 +1,80 @@
+- Flatpak
+- FlatHub
+
+## Installation Process
+
+1. Go to the Flathub website, type "Decoder" in the search bar, and click on **Install**.
+![Screenshot of the install button being highlighted by a red rectangle](images/01_decoder.png)
+
+1. Then, run the following command in your terminal
+`flatpak run com.belmoussaoui.Decoder`
+
+## How to Create a QR Code
+
+There are two types of QR codes you can create on Decoder. Choose the option that best suits your needs:
+
+- [Text](#text)
+- [Wifi](#wifi)
+
+### Text
+
+1. Click on the **Text** button
+![Screenshot of a red arrow pointing next to a grey button with the word, Scan written in black font](images/02_decoder-text.png)
+1. Add a link to your desired website and add a description
+![Screenshot of a red arrow pointing upwards the text box, containing the description "Check out this song" with a Youtube link next to it ](images/03_decoder-text.png)
+1. Click on **Create**
+![Screenshot of a red arrow pointing next to a grey button that has a QR code symbol and the word "Create" underneath it. ](images/04_decoder-text.png)
+1. Click on **Save**
+![Screenshot of a red arrow pointing next to a grey button that has the word "Save" written in black. ](images/05_decoder-text.png)
+1. Click on **Export**
+![Screenshot of a red arrow pointing next to a grey button that has the word "Export" written in black. ](images/06_decoder-text.png)
+
+### Wifi
+
+1. Click on the **Wifi** button
+![Screenshot of a red arrow pointing at a grey button that has the word "Wifi" written in black.](images/01_decoder-wifi.png)
+1. Add the name of your network, password, and pick its form of protection from the encryption algorithm dropdown menu
+![Screenshot the wifi's name, password, and encryption algorithm being displayed. ](images/02_decoder-wifi.png)
+1. Click on **Export**
+![Screenshot of a red arrow pointing at a grey button that has the word "Export" written in black.](images/03_decoder-wifi.png)
+1. Click on **Save**
+![Screenshot of a red arrow pointing at a grey button that has the word "Save" written in black.](images/04_decoder-wifi.png)
+
+!!! tip
+If your wifi network is hidden, click on the **Hidden** button.
+
+### How to Scan a QR code
+
+In addition to creating and generating QR codes, you can use Decoder to scan QR codes you saved on you computer. Here's how:
+
+1. Click on **Scan**
+![Screenshot of a red circle on a grey button that has the word "Scan" written in black.](images/01_decoder-scan.png)
+1. Click on  **Choose from a Screenshot**
+<!--- add a screenshot here --->
+1. Pick your desired effects and click on **Take a Screenshot**
+<!--- add a screenshot here --->
+1. Click on **Share**
+![Screenshot of a red arrow pointing at a blue button that has the word "Share" written in white.](images/04_decoder-scan.png)
+1. Scan the QR code with your mobile device
+
+!!! note
+ To scan a QR code straight from your computer, ensure that you grant the app access to your computer's camera.
+
```

Comment:
Again, I must refer you to [this document](https://docs.rockylinux.org/guides/contribute/rockydocs_formatting/). Your admonitions need to be formatted with a blank line between your admonition label and the content, and the content needs to be indented 4 spaces. (see "Admonitions" in the document link above). Your admonitions should look like this when complete:

```
!!! note

    To scan a QR code straight from your computer, ensure that you grant the application access to your computer's camera.
```
Note, too, that "app" above has been changed to "application." This is because our documentation is translated into other languages, and abbreviations, idioms, or slang, are problems. Always use clear wording that avoids these things. 

Thank you!